version

```diff
@@ -0,0 +1 @@
+0
```

Comment:
Please start with 1

Reply:
It starts with 0 in the commit so that when the commit set is merged to main, the workflow will increase it to 1.